src/components/LaunchDetailsPage/LaunchDetailsPage.tsx

```diff
@@ -8,10 +9,13 @@ import {
 } from '@mui/material';
 import { SecondaryButton } from '../../shared/components/SecondaryButton';
 import { LaunchCardContent } from './components/LaunchCardContent';
-import { launchDetails } from './mocks/launchDetails';
+import { getActiveSlideData } from './selectors/selectors';
+import type { LaunchDetails } from './types/LaunchDetails';
 import './LaunchDetailsPage.scss';
 
 export const LaunchDetailsPage = () => {
+  const details: LaunchDetails = useSelector(getActiveSlideData);
```

Comment:
It is depend on code style but we can use destructuring here

Reply:
I need the whole object to pass it into the prop in line 34